utils/utils.go

```diff
@@ -19,7 +20,15 @@ func OpenInBrowser(url string) error {
 	if err != nil {
 		return err
 	}
-	return run.PrepareCmd(browseCmd).Run()
+	err = run.PrepareCmd(browseCmd).Run()
+	if err != nil {
+		fmt.Printf("Error opening %s in your browser.\n", url)
+		envVar := os.Getenv("BROWSER")
+		if envVar != "" {
+			fmt.Printf("$BROWSER is set to %s. Is that correct?\n", envVar)
```

Comment:
1. Errors should be printed to stderr instead of to stdout
2. However, instead of printing to either stdout or stderr ourselves, could we instead wrap the original error and just add some more context to it? e.g.

```go
return fmt.Errorf("%w\nnote: check your BROWSER environment variable", err)
```